index.bs

```diff
@@ -92,8 +89,11 @@ is [=Registration=], where a [=scoped credential=] is created on an [=authentica
 with the present user's account (the account may already exist or may be created at this time). The second is
 [=Authentication=], where the [=[RP]=] is presented with an <em>[=Authentication Assertion=]</em> proving the presence
 and consent of the user who registered the [=scoped credential=]. Functionally, the [=Web Authentication API=] comprises
-two methods (along with associated data structures): {{makeCredential()}} and {{getAssertion()}}. The former is used
-during [=Registration=] and the latter during [=Authentication=].
+a {{ScopedCredential}} which extends the Credential Management API, and infrastructure which allows those credentials to be
```

Comment:
Added in https://github.com/w3c/webauthn/pull/384/commits/b1e87ca8f5feb2685dc9d47f033207e5972f86cc.